pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -2,20 +2,23 @@ package bootstrappolicy
 
 import (
 	"reflect"
+	"sort"
 	"testing"
 
 	"k8s.io/apiserver/pkg/authentication/serviceaccount"
 
 	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	scc "github.com/openshift/origin/pkg/security/securitycontextconstraints"
 )
 
 func TestBootstrappedConstraints(t *testing.T) {
-	expectedConstraints := []string{
+	// ordering of expectedConstraintNames is important, we check it against scc.ByPriority
+	expectedConstraintNames := []string{
 		SecurityContextConstraintPrivileged,
-		SecurityContextConstraintRestricted,
 		SecurityContextConstraintNonRoot,
 		SecurityContextConstraintHostMountAndAnyUID,
 		SecurityContextConstraintHostNS,
+		SecurityContextConstraintRestricted,
 		SecurityContextConstraintsAnyUID,
```

Comment:
With the latest logic fixes, `anyuid` is now actually first. Which sounds correct, it matches
```func (s ByPriority) Less(i, j int) bool {
[...]
        // a higher priority is considered "less" so that it moves to the front of the line
        if iSCCPriority > jSCCPriority {
                return true
        }
```